### ISSUE TITLE
Geomap included in shared tooltip

### ISSUE THREAD

**Issue by mbenmclean** [grafana/grafana#97776]
**Why is this needed**:
It would be very useful to have geomap included in shared tooltip so that the individual data points will show up on the map when hovering over another time-series plot. Unlike other time series plots, there is no visual time domain on a geomap. Often times the geomap data can be confusing and difficult to contextualize. Being able to hover over a timeseries plot at the time you are interested in and see where the corresponding data point is on the map would be incredibly useful. Here is an screenshot of my dashboard showing this feature is not supported. 

![Image](https://github.com/user-attachments/assets/10ab776c-6809-43ac-8077-a49da36ee927)

It would be very helpful to be able to slide the curser along the plot below the geomap and view the corresponding position in the the geomap.

**What would you like to be added**:
I want the geomap visualization to support shared tooltip so when I hover over a point on another timeseries plot the tooltip for the same point appears on the geomap

**Who is this feature for?**

Users of geomap visualization.

**Comment by nabelekt** [comment]
Agreed that this would be helpful! I'd also want the reverse: the ability to move the cursor between points on the geomap and have the tooltip jump around on the plots in a corresponding fashion.

**Comment by mbenmclean** [comment]
Yes agreed, I assumed that was a given, in that the likely implementation of this would support shared tooltip integration both ways. But yes you make a good point it should be a requirement that this works both ways.

**Comment by nabelekt** [comment]
Still relevant
